gbrl/learners/multi_gbt_learner.py

```
@@ -588,21 +588,30 @@ def plot_tree(self, tree_idx: int, filename: str, model_idx: Optional[int] = Non
         except RuntimeError as e:
             print(f"Caught an exception in GBRL: {e}")
 
-    def tree_shap(self, tree_idx: int, features:
-                  NumericalData,
-                  model_idx: Optional[int] = None) -> Union[np.ndarray, Tuple[np.ndarray, ...]]:
+    def tree_shap(self, tree_idx: int, features: NumericalData,
+                  model_idx: Optional[int] = None,
+                  return_base: bool = False) -> Union[np.ndarray, Tuple[np.ndarray, ...]]:
```

Ruff (0.15.21)

[warning] Boolean-typed positional argument in function definition

(FBT001)